packages/twenty-server/package.json

```diff
@@ -29,6 +29,7 @@
     "cache-manager-redis-yet": "^4.1.2",
     "class-validator": "patch:class-validator@0.14.0#./patches/class-validator+0.14.0.patch",
     "graphql-middleware": "^6.1.35",
+    "handlebars": "^4.7.8",
```

Comment:
why do we need this library? :)

Reply:
To resolve the data injection in the html template we provide to the payload

Reply:
got it!